lib/shared/messages.js

```diff
@@ -32,14 +32,18 @@ module.exports = {
    */
   info: {
 
-    flashComplete: _.template([
-      '<%= imageBasename %> was successfully written to',
-      '<%= drive.description %> (<%= drive.displayName %>)'
-    ].join(' ')),
+    flashComplete: (imageBasename, drive) => {
+      return [
+        `${imageBasename} was successfully written to`,
+        `${drive.description} (${drive.displayName})`
+      ].join(' ')
+    },
 
-    providePassword: _.template([
-      '<%= displayName %> needs special permissions to interact with raw devices. Please provide your password.'
-    ].join(' '))
+    providePassword: (displayName) => {
+      return [
+        `${displayName} needs special permissions to interact with raw devices. Please provide your password.`
+      ].join(' ')
```

Comment:
No need for the `.join()` here with only one string ;)

Reply:
Oh yes, right.

Reply:
Should I also remove enclosing array brackets?

Reply:
Made the changes.